projects/ng-swiper/src/lib/swiper/swiper.errors.ts

```
@@ -0,0 +1,33 @@
+  constructor(
+    message: string,
+    public options: SwiperOptions,
+    public id: string,
+    public swiper?: Swiper,
+    public error?: any,
+  ) {
+    const extraInfos = [
+      error ? `Error: ${error}` : null,
+      id ? `Id: ${id}` : null,
+    ].filter(info => info !== null);
+    super(`${message}${extraInfos?.length ? '\n' + extraInfos.join(',\n') : ''}`);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)